js/login.js

```diff
@@ -0,0 +1,16 @@
+const userLogin = document.querySelector("#login-btn")
+
+const setUserName = (name) => {
+    if(name){
+        userLogin.innerHTML = `Hello ${name}!`
+    }
+}
+
+let url = "https://staging-api.realdevsquad.com/users/self"
+
+fetch(url, {
+    "credentials": "include"
+})
+.then(res => res.json())
+.then(res => setUserName(res.first_name))
```

Comment:
Can the function name be a bit more descriptive? Like `setNameIfFound`?

Reply:
Done with the changes as per review. Thanks!